README.md

```diff
@@ -212,79 +244,51 @@ julia> pairwise(Euclidean(1e-12), x, x)
  0.0
 ```
 
-
 ## Benchmarks
 
-The implementation has been carefully optimized based on benchmarks. The script in `benchmark/benchmarks.jl` defines a benchmark suite
-for a variety of distances, under column-wise and pairwise settings.
+The implementation has been carefully optimized based on benchmarks. The script in
+`benchmark/benchmarks.jl` defines a benchmark suite for a variety of distances,
+under column-wise and pairwise settings.
 
-Here are benchmarks obtained running Julia 1.0 on a computer with a dual-core Intel Core i5-2300K processor @ 2.3 GHz.
-The tables below can be replicated using the script in `benchmark/print_table.jl`.
+Here are benchmarks obtained running Julia 1.5 on a computer with a quad-core Intel
+Core i5-2300K processor @ 3.2 GHz. Extended versions of the tables below can be
+replicated using the script in `benchmark/print_table.jl`.
 
-#### Column-wise benchmark
+### Column-wise benchmark
 
-The table below compares the performance (measured in terms of average elapsed time of each iteration) of a straightforward loop implementation and an optimized implementation provided in *Distances.jl*. The task in each iteration is to compute a specific distance between corresponding columns in two `200-by-10000` matrices.
+The table below compares the performance (measured in terms of average elapsed
+time of each iteration) of a straightforward loop implementation and an
+implementation specialized to `[Sq]Mahalanobis` provided in *Distances.jl*.
+The task in each iteration is to compute a specific distance between corresponding
+columns in two `200-by-10000` matrices.
```

Comment:
How about explaining that for other distances the implementation is equivalent to a loop?